Packages/Tracking Preview/PhysicsHands/Editor/PhysicsProviderEditor.cs

```diff
@@ -82,6 +84,34 @@ public override void OnInspectorGUI()
             serializedObject.ApplyModifiedProperties();
         }
 
+        private void WarningsSection()
+        {
+            // Solver iterations
+            if (Physics.defaultSolverIterations < 15)
```

Comment:
Would be nice to extract the value to a constant rather than using the same value in several places. Not going to block the approval though.